src/main/java/com/hotels/bdp/circustrain/bigquery/extraction/container/DeleteTableAction.java

```diff
@@ -33,8 +33,12 @@ public DeleteTableAction(Table table) {
   @Override
   public void run() {
     try {
-      table.delete();
-      log.debug("Deleted table '{}'", table.getTableId());
+      boolean deleted = table.delete();
+      if (deleted) {
+        log.debug("Deleted table '{}'", table.getTableId());
+      } else {
+        log.debug("Table not found: '{}'", table.getTableId());
```

Comment:
Since this might appear anywhere in the logs probably best to give this some context in the message - "Table not found for deletion"